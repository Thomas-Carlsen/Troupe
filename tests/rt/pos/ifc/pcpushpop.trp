let val x = 1 raisedTo {secret} 
    val c = pcpush authority 
    val _ = if x > 0 then let val _ = pcpop c in debugpc () end 
                     else () 
    (* val _ = pcpop c  *)
in debugpc ()
end