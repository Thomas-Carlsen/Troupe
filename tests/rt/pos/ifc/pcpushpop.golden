2019-12-11T23:34:08.350Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mPID:1c08ac4a-6980-4871-a2d5-538611de2a18@{}%{}    [39m [31mPC:{}               [39m [31mBL:{}               [39m 
[31mPID:1c08ac4a-6980-4871-a2d5-538611de2a18@{}%{}    [39m [31mPC:{}               [39m [31mBL:{}               [39m 
>>> Main thread finished with value: ()@{}%{}
